Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[workspace]
+resolver = "2"
+members = ["iroh-dns-server"]
```

Comment:
why a workspace, not just a top level crate?

Reply:
Historical reasons only - will move to top-level crate only, wanted to that too